### PR TITLE
fix: lint fixes

### PR DESCRIPTION
**Pull request by orrgottlieb**

### CODE REVIEW

**Review by sahariko — APPROVED** [review]

File: src/components/Menu/MenuItemButton/__stories__/menuItemButton.stories.js [hunk 1]
```diff
@@ -5,40 +5,40 @@ import Menu from "../../Menu/Menu";
 import { Invite } from "../../../Icon/Icons";
 
 export const menuItemButtonKindsTemplate = args => [
-  <Menu>
+  <Menu key="Primary">
```
Comment (hadasfa) [inline src/components/Menu/MenuItemButton/__stories__/menuItemButton.stories.js:8]: why?

File: src/components/Slider/SliderInfixHooks.jsx [hunk 2]
```diff
@@ -17,12 +17,12 @@ export function useSliderInfixComponent(kind) {
 
   const isPostfix = kind === INFIX_KIND.POSTFIX;
   if (indicateSelection && (isPostfix || ranged)) {
-    return [true, [], <SelectionIndicator kind={kind} />, { width: selectionIndicatorWidth }];
+    return [true, [], <SelectionIndicator key={kind} kind={kind} />, { width: selectionIndicatorWidth }];
```
Comment (hadasfa) [inline src/components/Slider/SliderInfixHooks.jsx:20]: is it not problematic that more than one component will have the same key?

File: src/components/Slider/SliderInfixHooks.jsx [hunk 3]
```diff
@@ -17,12 +17,12 @@ export function useSliderInfixComponent(kind) {
...
   }
   if (typeof infix === "object" && infix.icon) {
     const { icon, ...restIconProps } = infix;
     const iconProps = { ...defaultIconProps, ...restIconProps };
-    return [true, [], <Icon icon={icon} {...iconProps} />, {}];
+    return [true, [], <Icon key="infix-icon" icon={icon} {...iconProps} />, {}];
```
Comment (hadasfa) [inline src/components/Slider/SliderInfixHooks.jsx:25]: same key question

File: src/components/Toast/Toast.jsx [hunk 4]
```diff
@@ -53,14 +53,19 @@ const Toast = ({
...
       : null;
   }, [actions]);
   const toastButtons = useMemo(() => {
     return actions
       ? actions
           .filter(action => action.type === TOAST_ACTION_TYPES.BUTTON)
-          .map(({ type: _type, content, ...otherProps }) => <ToastButton {...otherProps}> {content} </ToastButton>)
+          .map(({ type: _type, content, ...otherProps }, index) => (
+            <ToastButton key={`alert-button-${index}`} {...otherProps}>
+              {" "}
```
Comment (hadasfa) [inline src/components/Toast/Toast.jsx:65]: i guess {" " } is by mistake

File: src/hooks/__tests__/useMergeRefs.jest.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (hadasfa) [inline src/hooks/__tests__/useMergeRefs.jest.js:1]: why?

**Review by hadasfa — APPROVED** [review]

**Pull request merged**